web/package.json

```diff
@@ -249,9 +249,9 @@
     "i18next-parser": "^9.0.2",
     "identity-obj-proxy": "^3.0.0",
     "is-interactive": "^2.0.0",
-    "jest": "^29.7.0",
     "jest-axe": "^9.0.0",
     "jest-chain": "^1.1.6",
```

Comment:
Do we still need all those jest dependencies?

Reply:
No, I removed them and that even resolved one more console warning, nice!